### DISCUSSION TITLE
Mermaid flowchart causes gitbook to error

### DISCUSSION THREAD

**Discussion by alongwill** [GitbookIO/community#806] [0. General bugs and questions]
When I try and create a Mermaid diagram starting with "flowchart"  on one of our pages, the error "Something went wrong" is shown, and my changes are lost. Using "graph" works without error.

The Mermaid added was:

```
flowchart TB
  %% Define Clusters
  subgraph GCP [Google Cloud]
    direction TB
    cp1["Control Plane Node 1"]
    cp2["Control Plane Node 2"]
    cp3["Control Plane Node 3"]
  end
```

UPDATE: Even this fails after a few minutes and shows "Something went wrong". Clicking on "Let us know what happened" doesn't work. I'm using Google Chrome.
<img width="338" alt="image" src="https://github.com/user-attachments/assets/d6466e3f-9831-4430-9488-a7e1afcd0fca">

**Comment by john-gitbook** [comment]
Hey Andy. Thanks for flagging this. It is a bug. I've raised a ticket to get this fixed. Will keep you posted on it.

**Reply by SimonGrefGK** [reply]
Hello John, I can only confirm that we also have the error. We would be very grateful if you could fix it as soon as possible, as a lot of process documentation depends on Mermaid. Thanks a lot.

**Reply by DanielSmithGK** [reply]
If it's any use: it is not limited to flowcharts only, got the same with sequence and gantt diagrams also. Happened in live edit as well as editing in a change request.

**Comment by john-gitbook** [comment]
@DanielSmithGK , @SimonGrefGK and @alongwill 

Thanks for your patience here. This should be fixed now. Can you take a look and let us know if there are still any issues?

**Reply by DanielSmithGK** [reply]
Hi @john-gitbook thanks for the info, I checked my previously malfunctioning mermaid blocks, they are working now, it seems fixed. Your quick help is appreciated!

**Reply by john-gitbook** [reply]
great. Thanks so much for confirming Daniel!
